utils/cli/src/google/build.ts

```diff
@@ -110,6 +110,15 @@ const build = async (id: string, opts: BuildOptions) => {
 
 	// Write package.json
 	await packageJson(metadata, opts.dir);
+
+	// Write metadata.scss
+	await fs.writeFile(
+		path.join(opts.dir, 'metadata.scss'),
```

Comment:
Should we save these into an `scss` folder?

Reply:
Since there are only two of them and they both end with scss (which distinguishes them as sass files), I thought it would be fine to keep them in the root.

Reply:
Yeah that's a fair point. I just think root is already cluttered as is so. 

Maybe we should rename `metadata.scss` to `mixins.scss` to reduce a migration step?